lib/origen_doc_helpers/pdf.rb

```diff
@@ -71,6 +75,7 @@ def generate_pdf
         end
       end
       cmd += "#{pdf_output_dir}/#{filename}"
+puts "cmd=#{cmd}"
```

Comment:
Probably didn't intend to keep this line

Reply:
Yeah -- fixed thanks